recipes/msys2/all/conanfile.py

```diff
@@ -60,81 +60,14 @@ class MSYS2Conan(ConanFile):
     def validate(self):
         if self.settings.os != "Windows":
             raise ConanInvalidConfiguration("Only Windows supported")
-        if tools.Version(self.version) >= "20210105" and self.settings.arch != "x86_64":
+        if self.settings.arch != "x86_64":
             raise ConanInvalidConfiguration("Only Windows x64 supported")
```

Comment:
It means that CCI recipes depending on msys2 won't support anymore building on 32 bits architecture.

Reply:
yes, but msys2/20210105 did not support 32bits either

Reply:
this is why I did not update msys2 in CCI recipes requiring it.

Reply:
ok, so users requiring 32bits can use msys2/20200517. What is the problem/what do you suggest ?

Reply:
build requirements can't be overridden. The problem is that we can't update msys2 version in CCI recipes depending on it, or we accept to explicitly drop support of 32 bits build machine in these recipes.

Reply:
You can use another msys2 installation by setting CONAN_BASH_PATH env var. Also, is it not possible to use cross-compilation profiles from x86_64 to x86 in this case, and use 64bits msys2 to compile 32bits libraries ?